POSTINSTALL-README.md

```diff
@@ -14,6 +15,15 @@ GRUB will prompt for a username and password. The default username is root.
 
 If you wish to password-protect booting existing entries, you can add the `grub_users root` entry in the specific configuration file located in the `/boot/loader/entries` directory.
 
+### Remove duplicate boot entries
```

Comment:
This applies to all rpm-ostree systems, not just secureblue. This document is for additional _hardening_, not fixing upstream issues. 

So please remove this section. That said, you can leave in the other unrelated formatting improvements.